lib/elixir_exif/tag.ex

```diff
@@ -51,10 +51,11 @@ defmodule ElixirExif.Tag do
     numerator = read_unsigned.(numerator) |> maybe_signed_int(signed)
     denominator = read_unsigned.(denominator) |> maybe_signed_int(signed)
     result = case {numerator, denominator} do
-      {numerator, 1} -> numerator
-      {1, denominator} -> "1/#{denominator}"
-      {numerator, denominator} -> numerator/denominator
-    end
+               {numerator, 1} -> numerator
```

Comment:
Well, „messed up” is a question of definition ;-) I aligned it to the case plus two spaces.